wnfs/README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <a href="https://github.com/wnfs-wg" target="_blank">
-    <img src="../assets/logo.png" alt="Fission Logo" width="100" height="100"></img>
+    <img src="https://raw.githubusercontent.com/wnfs-wg/rs-wnfs/main/assets/logo.png" alt="WNFS Logo" width="100" height="100"></img>
```

Comment:
@appcypher any reason not to go relative?

Reply:
Because it breaks on [crates.io](https://crates.io/crates/wnfs) since the parent workspace isn't published.

Reply:
@appcypher derp of course. my bad.